manim/mobject/mobject.py

```diff
@@ -1344,8 +1344,9 @@ def __init__(self, *mobjects, **kwargs):
 
 
 class _AnimationBuilder:
-    def __init__(self, mobject, generate_target=True):
+    def __init__(self, mobject, overridden_animations, generate_target=True):
         self.mobject = mobject
+        self.overridden_animations = []
```

Comment:
This should be `self.overridden_animations = overridden_animations`, correct?

Reply:
Indeed, it should -- good catch! ccfce05b